CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.16)
+project(evm_bridge_contracts)
+add_subdirectory (solidity)
```

Comment:
Put this line down lower in the file after the find_packages.
Also, immediately following it, you can add `add_subdirectory(antelope_contracts)`.

Reply:
Refactored folder structure, this should have been resolved.